Dockerfile

```diff
@@ -6,9 +6,6 @@ COPY . .
 RUN go build -o ./openfga ./cmd/openfga
 
 FROM alpine as final
-EXPOSE 8081
```

Comment:
These aren't needed. Without them `docker run -p 8080:8080 openfga/openfga run` runs just fine.

Since these ports are configurable anyways, it doesn't make sense to statically expose them.

Reply:
[EXPOSE doesn't actually publish the port](https://docs.docker.com/engine/reference/builder/#expose), but I think it is useful for documentation purposes if by default these are the ports OpenFGA is running on. In that sense, I would prefer to keep them. WDYT?